src/ControlSystem/Averager.hpp

```diff
@@ -45,6 +45,27 @@ class Averager {
   /// whether to return an averaged value for the 0th derivative piece of the
   /// function.
   Averager(double avg_timescale_frac, bool average_0th_deriv_of_q) noexcept;
+
+  // Explicitly defined move constructor due to the fact that the std::deque
+  // move constructor is not marked noexcept
+  Averager(Averager&& A) noexcept;
+  Averager& operator=(Averager&& A) noexcept {
+    if (this != &A) {
+      avg_tscale_frac_ = std::move(A.avg_tscale_frac_);
+      average_0th_deriv_of_q_ = std::move(A.average_0th_deriv_of_q_);
+      averaged_values_ = std::move(A.averaged_values_);
+      boost_none_ = std::move(A.boost_none_);
+      times_ = std::move(A.times_);
+      raw_qs_ = std::move(A.raw_qs_);
+      weight_k_ = std::move(A.weight_k_);
+      tau_k_ = std::move(A.tau_k_);
+    }
+    return *this;
+  }
+  Averager(const Averager&) = delete;
```

Comment:
This class used to be copyable, are you sure you don't want it to be anymore? I think the `default` copy semantics would work fine if you needed them. (Note: Not a change request, just confirming this is what you want :) )